suite-common/bluetooth/src/bluetoothSelectors.ts

```diff
@@ -46,3 +46,25 @@ export const prepareSelectAllDevices = <T extends BluetoothDeviceCommon>() =>
 
 export const selectScanStatus = <T extends BluetoothDeviceCommon>(state: WithBluetoothState<T>) =>
     state.bluetooth.scanStatus;
+
+export const selectAutoConnectPolicy = <T extends BluetoothDeviceCommon>(
+    state: WithBluetoothState<T>,
+) => state.bluetooth.autoConnectPolicy;
+
+export const selectConnectableDevices = <T extends BluetoothDeviceCommon>(
```

Comment:
What about calling it `selectNearbyConnectableDevices` or `selectAutoConnectableDevices`?